.github/workflows/pull-components.yml

```diff
@@ -26,7 +26,13 @@ jobs:
       - name: Create or reset ${{ env.PR_BRANCH }}
         working-directory: "bottle-components-repository"
         run: |
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
           git checkout -B ${{ env.PR_BRANCH }}
+          mkdir patch_dir
+          touch patch_dir/dummy
+          git add patch_dir/dummy
+          git commit -m "Add dummy file"
```

Comment:
Is this a test left in the final version?

Reply:
Yes: as git doesn't allow committing an empty directory, a dummy file need to be created.

Reply:
I think the standard practice is to name the file `.gitkeep`!